OnlineStoreTCA/Products/ProductList/ProductListView.swift

```diff
@@ -32,9 +32,13 @@ struct ProductListView: View {
                                     action: ProductListDomain.Action
                                         .product(id: action:)
                                 )
-                            ) {
-                                ProductCell(store: $0)
-                            }
+                            ) { cellStore in
+                                NavigationLink {
+                                    ProductDetailsView(store: cellStore)
```

Comment:
@pitt500 here^

Reply:
What is the problem here?

Reply:
the `cellStore` here is type of `ProductDomain` while we want to use `ProductDetailsViewDomain`.

Reply:
Let me take a look